### PR TITLE
reorganize admin dash

### PR DESCRIPTION
**Pull request by vincanger**
Implements part of #149 

- [x] organize files vertically
- [x] update docs
- [x] update app_diff 
- [x] make sure opensaas-sh PLAUSIBLE_BASE_URL has appened '/api' path

### CODE REVIEW

**Review by Martinsos — COMMENTED** [review]
Looking good in general! I am wondering if you put too much stuff under admin. Sometimes it is not easy to say where something should go though. So, what does "admin" mean for us -> functionality that is accessible only by admins, right? If the app didn't have concept of admin user, then these functionalities could probably be deleted.

For example fonts, are those specific only for admin part really? If so, ok!
Messages thing -> that is also specific only for Admin?
I think we have some UI components that we put in open-saas so people can easily use them, but they are nto specific for Admin -> however we do show them in Admin. In that case, I would consider actually taking them out of Admin even though they are used there, make them standalone. Put them under sometihng like `common/components/`.

Then there is analytics. You grouped it all under `analytics/` which is great. But I wonder if it should be standalone. Because analytics is not neccessarily specific to admin. It is analytics, we collect them. Sure, only admin can check them, but even if there was no admin we would maybe check them somehow in the database? Ha, not sure. Maybe to belong under admin. What is your reasoning about analytics?

**Comment by vincanger** [comment]
> Then there is analytics. You grouped it all under `analytics/` which is great. But I wonder if it should be standalone. Because analytics is not neccessarily specific to admin. It is analytics, we collect them. Sure, only admin can check them, but even if there was no admin we would maybe check them somehow in the database? Ha, not sure. Maybe to belong under admin. What is your reasoning about analytics?

The thing is, all the components and logic within analytics is only being used in the admin dashboard. When I extract this dir, for example, I end up with some imports to other smaller components that are still within the admin dashboard, and are also only being used there. This makes me think that it's current, and possible only, home should be `admin`. But, on the other hand, analytics itself is already kind of big, so it might be worth separating it. WDYT?

File: template/app/main.wasp [hunk 1]
```diff
@@ -420,4 +292,136 @@ entity File {=psl
...
+  id                       Int             @id @default(autoincrement())
+  createdAt                DateTime        @default(now())
+
+  message                  String
+  level                    String
+psl=}
+
+// TODO: add functionality to allow users to send messages to admin
+// and make them accessible via the admin dashboard
+entity ContactFormMessage {=psl
```
[the remark this answers is not in the document]
Comment (vincanger) [reply template/app/main.wasp:416]: Moved it to its own little home

File: template/app/src/admin/settings/SettingsPage.tsx
[the remark this answers is not in the document]
Comment (vincanger) [reply template/app/src/admin/settings/SettingsPage.tsx:1]: Yeah its just an extra component for changing user settings in the admin panel

**Comment by Martinsos** [comment]
Discussed separately: we will try to split a part of analytics out.

**Review by Martinsos — APPROVED** [review]
LGTM!

**Pull request merged**